Formula/teleport.rb

```diff
@@ -0,0 +1,47 @@
+class Teleport < Formula
+  desc "Modern SSH server for teams managing distributed infrastructure"
+  homepage "https://gravitational.com/teleport"
+  url "https://github.com/gravitational/teleport/archive/v1.3.2.tar.gz"
+  sha256 "2f2f5aa019c99d1a829f16745aa6d81778945ca29b3e936cc9117aac832e12e1"
+
+  depends_on "go" => :build
+  depends_on "godep" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath / "src/github.com/gravitational/teleport").install buildpath.children
+    cd "src/github.com/gravitational/teleport" do
+      system "godep", "restore"
+      system "go", "build", "-o", bin / "tctl", "-ldflags", "-w -s", "./tool/tctl"
+      system "go", "build", "-o", bin / "teleport", "-ldflags", "-w -s", "./tool/teleport"
+      system "go", "build", "-o", bin / "tsh", "-ldflags", "-w -s", "./tool/tsh"
+      prefix.install_metafiles
+      cd "web/dist" do
+        system "zip", "-q", "-r", "webassets.zip", "."
+        system "cat webassets.zip >> #{bin}/teleport"
```

Comment:
changing it to `system "make", "release", "BINDIR=#{bin}"` i couldn't get past this.

```
==> godep restore
==> make release BINDIR=/usr/local/Cellar/teleport/1.3.2/bin
rm -rf build
/Library/Developer/CommandLineTools/usr/bin/make -f version.mk setver
go build -o build/teleport -i -ldflags '-w -s' ./tool/teleport
rm -rf teleport
go build -o build/tctl -i -ldflags '-w -s' ./tool/tctl
go build -o build/tsh -i -ldflags '-w -s' ./tool/tsh
rm -rf build/app
rm -f web/dist/app/app
cp README.md build
fatal: Not a git repository (or any of the parent directories): .git
go install github.com/gravitational/teleport/tool/tsh: mkdir build/: not a directory
make: *** [tsh] Error 1
make: *** Waiting for unfinished jobs....
go install github.com/gravitational/teleport/tool/tctl: mkdir build/: not a directory
make: *** [tctl] Error 1
go install github.com/gravitational/teleport/tool/teleport: mkdir build/: not a directory
make: *** [teleport] Error 1```